src/components/dashboard/styles/index.css

```diff
@@ -437,3 +437,12 @@ nav {
 }
 
 
+.viewdetailslink a:hover {
```

Comment:
this is not readable use hyphens

Reply:
Do mind shading more light on what you mean by not readable and how the hyphens will help with making it readable

Reply:
something like this `.view-details-link` would be better. I understand naming is hard, but let's try to stick to convention.